Doc/whatsnew/3.13.rst

```diff
@@ -521,6 +521,15 @@ io
   built on debug mode <debug-build>`.
   (Contributed by Victor Stinner in :gh:`62948`.)
 
+inspect
+-------
+
+* Add *bound_arg* parameter to :func:`inspect.Signature.from_callable`
+  and :func:`inspect.signature`: keep the ``self`` parameter
+  in the method signature if *bound_arg* is True.
```

Comment:
Reword here to match the updated main function docs (including mentioning the getfullargspec compatibility)